### PR TITLE
Hero Card Info

### PR DESCRIPTION
**Pull request by rafaelmakaha**
This Pull Request adds a card with hero description.

### CODE REVIEW

File: src/pages/Hero/styles.ts [hunk 1]
```diff
@@ -1,5 +1,31 @@
 import styled from 'styled-components'
+import { theme } from '../../styles'
 
 export const Content = styled.div`
+    
+`
 
+export const InfoCard = styled.div`
+    align-items: flex-start;
+    background-color: ${theme.background2};
```
Comment (wesleyadriann) [inline src/pages/Hero/styles.ts]: utilizar a prop do theme
```suggestion
    background-color: ${(props) => props.theme.background2};
```

File: src/pages/Hero/styles.ts [hunk 2]
```diff
@@ -1,5 +1,31 @@
 import styled from 'styled-components'
+import { theme } from '../../styles'
```
Comment (wesleyadriann) [inline src/pages/Hero/styles.ts]: deletar
```suggestion

```

File: src/pages/Hero/styles.ts [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    padding: 16px 16px 0 16px;
+    position: absolute;
+    width: 40%;
+`
+
+export const Text = styled.p`
+    color: ${theme.colors.black};
```
Comment (wesleyadriann) [inline src/pages/Hero/styles.ts]: utilizar a prop do theme
```suggestion
    color: ${(props) => props.theme.colors.black};
```

File: src/pages/Hero/styles.ts [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+    line-height: 2vh;
+
+    strong {
+        color: ${theme.colors.primaryRed};
```
Comment (wesleyadriann) [inline src/pages/Hero/styles.ts]: utilizar a prop do theme
```suggestion
        color: ${(props) => props.theme.colors.primaryRed};
```

File: src/styles/theme.ts [hunk 5]
```diff
@@ -3,6 +3,7 @@ export const theme = {
   fontFamily: 'Roboto',
   fontWhite: '#e2dace',
   background: '#13120e',
+  background2: '#204C39',
```
Comment (wesleyadriann) [inline src/styles/theme.ts]: utilizar nome descritivo ou mudar para key colors e descrever a cor
```suggestion
  cardBackground: '#204C39',
```

File: src/pages/Hero/styles.ts [hunk 6]
```diff
@@ -1,5 +1,31 @@
...
+    flex-direction: column;
+    gap: 16px;
+    padding: 16px 16px 0 16px;
+    position: absolute;
+    width: 40%;
+`
+
+export const Text = styled.p`
+    color: ${theme.colors.black};
+    line-height: 2vh;
```
Comment (wesleyadriann) [inline src/pages/Hero/styles.ts]: line-height de 2 viewport ?

File: src/pages/Hero/styles.ts [hunk 7]
```diff
@@ -1,5 +1,31 @@
...
+
+    strong {
+        color: ${theme.colors.primaryRed};
+    }
+
+    li {
+        margin: 2vw;
```
Comment (wesleyadriann) [inline src/pages/Hero/styles.ts]: margin de 2 viewport ?

**Review by wesleyadriann — CHANGES_REQUESTED** [review]

File: src/pages/Hero/styles.ts (hunk 1 above)
Comment (rafaelmakaha) [reply src/pages/Hero/styles.ts]: Done

File: src/pages/Hero/styles.ts (hunk 2 above)
Comment (rafaelmakaha) [reply src/pages/Hero/styles.ts]: Done

File: src/pages/Hero/styles.ts (hunk 3 above)
Comment (rafaelmakaha) [reply src/pages/Hero/styles.ts]: Done

File: src/pages/Hero/styles.ts (hunk 4 above)
Comment (rafaelmakaha) [reply src/pages/Hero/styles.ts]: Done

File: src/styles/theme.ts (hunk 5 above)
Comment (rafaelmakaha) [reply src/styles/theme.ts]: Done

File: src/pages/Hero/styles.ts (hunk 7 above)
Comment (rafaelmakaha) [reply src/pages/Hero/styles.ts]: Changed to px, something went wrong with my editor.

File: src/pages/Hero/styles.ts (hunk 6 above)
Comment (rafaelmakaha) [reply src/pages/Hero/styles.ts]: Removed. It wasn't adding any value

**Pull request merged**